Source/Public/New-ADOPSElasticpool.ps1

```diff
@@ -0,0 +1,44 @@
+function New-ADOPSElasticPool {
+    [CmdletBinding()]
+    param (
+        [Parameter(Mandatory)]
+        [string]$PoolName,
+        
+        # Request body found at https://docs.microsoft.com/en-us/rest/api/azure/devops/distributedtask/elasticpools/create?view=azure-devops-rest-7.1
+        [Parameter(Mandatory)]
+        [string]
+        $Body,
+
+        [string]$Organization,
+        
+        [Parameter()]
+        [guid]
+        $ProjectId,
+
+        [Parameter()]
+        [boolean]
+        $AuthorizeAllPipelines = $false,
+
+        [Parameter()]
+        [boolean]
+        $AutoProvisionProjectPools = $false
+
+    )
+
+    if (-not [string]::IsNullOrEmpty($Organization)) {
+        $Org = GetADOPSHeader -Organization $Organization
+    } else {
+        $Org = GetADOPSHeader
+        $Organization = $Org['Organization']
+    }
+
+    if ($PSBoundParameters.ContainsKey('ProjectId')) {
+        $Uri = "https://dev.azure.com/$Organization/_apis/distributedtask/elasticpools?poolName=$PoolName`&authorizeAllPipelines=$AuthorizeAllPipelines`&autoProvisionProjectPools=$AutoProvisionProjectPools&projectId={projectId}&api-version=7.1-preview.1"
```

Comment:
I believe the `{projectID}` should be replaced with `$ProjectID` here?

Reply:
Thanks, good find :) Sometimes it goes to quick!